src/main/java/lotto/domain/LottoNumber.java

```diff
@@ -1,21 +1,35 @@
 package lotto.domain;
 
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Objects;
 
 public class LottoNumber {
     protected static final int MIN_NUMBER = 1;
     protected static final int MAX_NUMBER = 45;
+    private static final Map<Integer, LottoNumber> cache = new HashMap<>();
```

Comment:
```suggestion
    private static final Map<Integer, LottoNumber> cache =
        IntStream.rangeClosed(MIN_NUMBER, MAX_NUMBER)
                    .mapToObj(LottoNumber::new)
                    .collect(Collectors.toMap(lottoNumber -> lottoNumber.number, Function.identity()));
```
이런 식으로 선언과 동시에 초기화가 가능합니다 :)

Reply:
어떻게 할까 살짝 고민했었는데, 제안해주신 방향으로 수정했습니다~!
감사합니다~!~! 👍